### DISCUSSION TITLE
PatchTool 7.11 12/08

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#495] [Announcements]
7.11相容性更新、更新7.05漢化、修正部分翻譯。

字體問題預計再找時間處理。

以下兩種方法擇一使用即可；請從release下方連結或內文連結下載檔案。

## 方法一：直接覆蓋遊戲檔案

解壓縮後直接覆蓋`SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\sqpack\ffxiv`內的同名檔案即可。內含三個000000開頭的字體相關檔案與三個0a0000開頭的文本相關檔案。

設定：Windows作業系統，非高清字體，以繁體中文覆蓋日文和英文，官方渠道版本。

設定相同的人可以考慮直接下載這包，（可能）比較不會遇到問題。其他作業系統未測試。

- ffxiv-20241208.rar：請從下方連結下載。

## 方法二：使用漢化器

使用方法請參照[readme](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/2.-%E4%BD%BF%E7%94%A8)。

檔案：

- FFXIVChnTextPatch-20241208.rar：漢化器本體。需要自行調整設定的人請下載這包手動漢化。請從下方連結下載。

視需求選用：

- [font-custom-20241005.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.3.1.0/font-custom-20241005.rar)：高畫質字體包，請取代原本的resource/font資料夾。請參閱[詳細使用說明](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。
- rawexd-zhcn-20241208.rar：簡體中文CSV，請取代原本resource/rawexd資料夾。
- backup-20241208.rar：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。只適用於這個release發布時的遊戲版本。

※若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。更多資訊可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/'>PatchTool 7.11 12/08</a>.</em>

**Comment by loier3600** [comment]
請問.....老主顧點開交貨的腳本似乎會卡住?

**Comment by jc-ff14** [comment]
請問一下MAC book 使用 12/08版本的漢化, 無法顯示現在最新的莫古莫古大收集商店頁面, 請問該如何解決?

**Comment by qihao913** [comment]
更新後20250326 不能用了...能幫忙支援更新嗎?.. 感恩

**Comment by Paul332300** [comment]
目前 7.2 版還可使用，但 ID 顯示異常。
![螢幕擷取畫面 2025-03-25 231040](https://github.com/user-attachments/assets/2607d701-6275-4f6b-9d41-a77a48fa773b)
![螢幕擷取畫面 2025-03-25 231202](https://github.com/user-attachments/assets/33f3abac-92e2-4d38-80b6-761e5889cd92)

**Reply by qihao913** [reply]
我用這個 ffxiv-20241208.rar 替換了6個檔案後，登入過程就會被閃退
重新安裝遊戲後，還是一樣.
請問，有什麼地方沒注意到的嗎?..
目前版本是2025.03.10.0000.00001(11500960) - 4.2.0

**Reply by ShingHK** [reply]
替換檔這個已經不能用了，只能等大大更新，如果要漢化就只能用漢化器，但是只有7.05的，7.05後的都是沒有漢化的

**Reply by qihao913** [reply]
謝謝回覆，等 大大  更新了．
